README.md

```diff
@@ -133,6 +133,34 @@ const App = () => {
 };
 ```
 
+### [Zod](https://github.com/vriad/zod)
```

Comment:
shall we move this on top of Joi please (or even superstruct)?

Reply:
Done ✅

Reply:
thank you